third_party/WebKit/Source/modules/webcl/WebCLOpenCL.cpp

```diff
@@ -2,16 +2,16 @@
 // Use of this source code is governed by a BSD-style license that can be
 // found in the LICENSE file.
 
-#include "wtf/build_config.h"
-
 #include "modules/webcl/WebCLOpenCL.h"
 
+#include "wtf/build_config.h"
+
 #if OS(ANDROID)
 #include <dlfcn.h>
 #endif
-#include <string.h>
+#include "wtf/CPU.h"
```

Comment:
I can't find any rational explanation why you put it after `#include <dlfcn.h>`.